.env.example

```
@@ -37,9 +37,9 @@ AI_GATEWAY_API_KEY=your_ai_gateway_api_key  # Get from https://vercel.com/dashbo
 
 # Individual provider keys (used when AI_GATEWAY_API_KEY is not set)
 ANTHROPIC_API_KEY=your_anthropic_api_key  # Get from https://console.anthropic.com
-OPENAI_API_KEY=your_openai_api_key  # Get from https://platform.openai.com (GPT-5)
+OPENAI_API_KEY=your_openai_api_key  # Get from https://platform.openai.com (GPT-5.2 Codex)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)